src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodesDefinition.xml

```diff
@@ -64,16 +64,41 @@
   
   <Node Name="Database" LocLabel="string.Empty" Image="Database" BaseClass="ModelBased" NodeType="Database" IsAsyncLoad="" Strategy="CreateModel" TreeNode="ExpandableSchemaTreeNode" PutFoldersAfterNodes="true">
     <ConditionalChildQuerierType Name="SqlSchema" SettingsFlag="GroupBySchema"/>
+    <Filters>
+      <!-- 
+        Excluding built-in schemas for backward compatibility from the database node and moving them to legacy schema folder.
+        The list of schemas was obtained from: 
+        https://learn.microsoft.com/en-us/sql/relational-databases/security/authentication-access/ownership-and-user-schema-separation?view=sql-server-ver16#built-in-schemas-for-backward-compatibility
+      -->
+      <Or>
+        <Filter Property="IsSystemObject" Value="0" Type="bool" />
+        <Filter Property="Name" Value="dbo" Type="string"/>
```

Comment:
For future-proofing I would suggest going back to a hardcoded list of the db_* ones we want to ignore. If a new system schema is added in the future this logic will currently result in it going into the builtin schemas folder which would be annoying to have to notice and remember to fix.